src/main/java/mcscheduler/logic/commands/CancelLeaveCommand.java

```diff
@@ -105,6 +103,19 @@ public CommandResult execute(Model model) throws CommandException {
                 MESSAGE_CANCEL_LEAVE_SUCCESS, workerIndexes.size(), assignStringBuilder.toString()));
     }
 
+    private String printOutOfBoundsWorkerIndexError(Index workerIndex) {
+        return String.format(Messages.MESSAGE_INVALID_COMMAND_FORMAT,
+                String.format(Messages.MESSAGE_INVALID_WORKER_DISPLAYED_INDEX, workerIndex.getOneBased())
+                        + MESSAGE_USAGE);
+    }
+
+    private String printOutOfBoundsShiftIndexError(Index shiftIndex) {
+        return String.format(Messages.MESSAGE_INVALID_COMMAND_FORMAT,
+                String.format(Messages.MESSAGE_INVALID_SHIFT_DISPLAYED_INDEX, shiftIndex.getOneBased())
+                        + MESSAGE_USAGE);
+    }
```

Comment:
Consider abstracting these methods to `CommandUtil` or some kind of util class as it appears quite a few times.

Reply:
The MESSAGE_USAGE is unique to each command and I don't want to pass in the Class object as a parameter into the function

Reply:
Actually can remove the `%1$` in `Invalid command format\\n%1$s` so we can just use normal string concatenation but might affect all functions using `MESSAGE_INVALID_COMMAND_FORMAT`

Reply:
I changed the printOutOfBoundsShiftIndexError to take in an extra String parameter `messageUsage`